src/pyramid/config/routes.py

```diff
@@ -40,12 +40,9 @@ def add_route(
         inherit_slash=None,
         **predicates
     ):
-        """ Add a :term:`route configuration` to the current
-        configuration state, as well as possibly a :term:`view
-        configuration` to be used to specify a :term:`view callable`
-        that will be invoked when this route matches.  The arguments
-        to this method are divided into *predicate*, *non-predicate*,
-        and *view-related* types.  :term:`Route predicate` arguments
```

Comment:
Unrelated to the current PR, but thought it was easier to cleanup that to open a separate ticket.  Easy to revert if you request so!